src/main/scala/io/iohk/ethereum/domain/Transaction.scala

```diff
@@ -73,7 +99,7 @@ case class TransactionWithAccessList(
     receivingAddress: Option[Address],
     value: BigInt,
     payload: ByteString,
-    accessList: List[AccessListItem]
+    accessList: Option[List[AccessListItem]]
```

Comment:
why introducing an option here? is it for the field not being encoded if the list is empty?